lib/cogctl/actions/relays/delete.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(_options, args,  _config, endpoint),
+    do: with_authentication(endpoint, &do_delete(&1, args))
+
+  defp do_delete(_endpoint, []) do
+    display_arguments_error
+  end
+
+  defp do_delete(endpoint, relay_names) when is_list(relay_names) do
+    Enum.reduce_while(relay_names, :ok, fn relay_name, _acc ->
+      case do_delete(endpoint, relay_name) do
+        :ok ->
+          {:cont, :ok}
+        :error ->
+          {:halt, :error}
+      end
+    end)
+  end
+
+  defp do_delete(endpoint, relay_name) do
+    case CogApi.HTTP.Client.relay_delete(%{name: relay_name}, endpoint) do
+      :ok ->
+        display_output("Deleted #{relay_name}")
```

Comment:
Ah, there are side-effects we're relying on during the `reduce_while` call above.

Is there a reason we would stop deleting relays if we encountered a "bad" one (the others might be fine)? In that case, it seems like `Enum.each` would be a better conceptual fit, since that's intentionally invoked for side-effects.